src/otx/algorithms/classification/adapters/mmcls/datasets/otx_datasets.py

```diff
@@ -329,6 +327,19 @@ def load_annotations(self):
             self.gt_labels.append(class_indices)
         self.gt_labels = np.array(self.gt_labels)
 
+        self._update_heads_information()
+
+    def _update_heads_information(self):
+        """Update heads information to find the empty heads.
+
+        If there are no annotations at a specific head, this should be filtered out to calculate loss correctly.
+        """
+        num_cls_heads = self.hierarchical_info["num_multiclass_heads"]
+        for head_idx in range(num_cls_heads):
+            labels_in_head = self.gt_labels[:, head_idx]  # type: ignore[call-overload]
+            if max(labels_in_head) < 0:
+                self.hierarchical_info["empty_multiclass_head_indices"].append(head_idx)
+
```

Comment:
It's weird to see "head" information in "dataset". Let's change to other terms in the future. Maybe, "empty_multiclass_label_indices"?

Reply:
Agree, I think it could be done at the refactoring phase to unify Multi-class / Multi-label / H-label Classification